event_model/__init__.py

```diff
@@ -534,16 +534,17 @@ def fill_event(self, doc, include=None, exclude=None, inplace=None):
                     datum_doc = self._datum_cache[datum_id]
                 except KeyError as err:
                     err_with_key = UnresolvableForeignKeyError(
```

Comment:
not a merge stopping comment but is there a reason why you didnt do just
``raise UnresolvableForeignKey(datum_id, ...)``

as you did below? it is probably a little neater

Reply:
Not sure what I was thinking. Thanks for pointing it out. Will push a fix.